styles/planet-minecraft/catppuccin.user.css

```
@@ -128,6 +128,30 @@
       }
     }
 
+    .content-actions {
+          li {
+              a {
+                  color: @text !important;
+              }
+              
+              a:hover, .js_link:hover, .link:hover {
+                  background-color: @surface2 !important;
+                  color: @text !important;
+              }
+              
+              .material-icons {
+                  color: @subtext1 !important;
+              }
+              
+              background: @surface0 !important;
+              border-color: @surface0 !important;
```

Most times with nested CSS the lines for this selector go first, and then the nested stuff.
```suggestion
              background: @surface0 !important;
              border-color: @surface0 !important;

              a {
                  color: @text !important;
              }
              
              a:hover, .js_link:hover, .link:hover {
                  background-color: @surface2 !important;
                  color: @text !important;
              }
              
              .material-icons {
                  color: @subtext1 !important;
              }
```